pkgs/by-name/zo/zotero-beta/package.nix

```diff
@@ -2,104 +2,78 @@
 , stdenv
 , fetchurl
 , wrapGAppsHook3
-, autoPatchelfHook
 , makeDesktopItem
+, alsa-lib
 , atk
 , cairo
 , coreutils
-, curl
-, cups
-, dbus-glib
 , dbus
+, dbus-glib
 , dconf
-, fontconfig
-, freetype
 , gdk-pixbuf
 , glib
-, glibc
+, gnome
+, gsettings-desktop-schemas
 , gtk3
+, libGL
 , libX11
```

Comment:
Consider getting these under `xorg` instead of at the top level. For example, `libX11` should be `xorg.libX11`